.dagger/mage/engine.go

```diff
@@ -21,6 +21,10 @@ var (
 	EngineContainerName = distconsts.EngineContainerName
 )
 
+func shouldLeaveOldEngine() bool {
+	return os.Getenv("DAGGER_LEAVE_OLD_ENGINE") == "1"
```

Comment:
Any value other than "0" and "" should count as true

Reply:
`”false”` should be count as `false`?

Reply:
> ”false” should be count as false?

Yes, good idea